Project 2/styles.css

```diff
@@ -0,0 +1,103 @@
+section {
+    background-color: #fff7e5;
+    max-width: 640px;
+    margin: 0 auto;
+    padding: 20px 30px;
+    border-radius: 80px;
+    border: solid rgb(255, 211, 226) 2px;
+
+}
+
+body {
+    color: rgb(44, 59, 50);
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    line-height: 40px;
```

Comment:
I don't recommend setting a `line-height` using px, especially not on the body, as it can have unexpected impact on any text that is _larger_ than this. A unit-less rule (e.g., `line-height: 1.3;`) wouldn't have this issue because it will adapt based on the font size.